### DISCUSSION TITLE
Unable to initialize device. Windows 10

### DISCUSSION THREAD

**Discussion by andrey1887** [libusb/libusb#1474] [Q&A] [answered]
Hello. Help me please to solve this issue. I have installed libusb for using it with libmtp on MSYS2 / MinGW (Windows 10 x64). But when i run command mtp-detect in MinGW console, the next message appears:

``` 
libmtp version: 1.1.21

Listing raw device(s)
Device 0 (VID=0bb4 and PID=0c93) is a HTC EVO 4G LTE/One V (ID1).
   Found 1 device(s):
   HTC: EVO 4G LTE/One V (ID1) (0bb4:0c93) @ bus 1, dev 7
Attempting to connect device(s)
libusb_open() failed!: No error
LIBMTP PANIC: Unable to initialize device
Unable to open raw device 0
OK. 
```
What am I doing wrong ?

**Comment by mcuee** [comment, marked as answer]
Have you install supported driver like WinUSB for your device?
https://github.com/libusb/libusb/wiki/Windows#user-content-Driver_Installation

**Reply by andrey1887** [reply]
Thank you, @mcuee. After installing WinUSB by Zadig it works fine.
